src/irj_parser/src/irj_ast.ml

```diff
@@ -0,0 +1,74 @@
+(* Copyright Inria, contributors: Raphaël Monat <raphael.monat@lip6.fr> (2019)
+   Mathieu Durero <mathieu.durero@dgfip.finances.gouv.fr>, David Declerck (2023)
+
+   This program is free software: you can redistribute it and/or modify it under
+   the terms of the GNU General Public License as published by the Free Software
+   Foundation, either version 3 of the License, or (at your option) any later
+   version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT
+   ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+   FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+   details.
+
+   You should have received a copy of the GNU General Public License along with
+   this program. If not, see <https://www.gnu.org/licenses/>. *)
+
+type t = { pos_filename : string; pos_loc : Lexing.position * Lexing.position }
```

Comment:
That type should not be named `t` as `t`would be the main type of this module, which we expect to be the AST of a whole test file and not a code position.